hack/llm-operator-values.yaml

```diff
@@ -32,6 +34,12 @@ global:
     secretAccessKeyKey: secretAccessKey
 
 
+dex-server:
+  database:
+    # TODO(kenji): Find a way to make Dex obtain the password from a k8s secret.
+    password: ps_password
```

Comment:
Not an ideal way, but what I thought of is using envsubst in initContainer to generate a configuration file and get the password from env.